bin/make-consensus.py

```diff
@@ -181,13 +200,25 @@ def main():
         sample = result.stdout.strip()
 
     consensusFile = join(tempdir, 'consensus.fasta')
-    result = e.execute(
-        "bcftools consensus --sample '%s' --iupac-codes %s --fasta-ref "
-        "'%s' '%s' > '%s'" %
-        (sample, maskArg, args.reference, vcfFile, consensusFile))
 
-    consensus = list(FastaReads(consensusFile))[0]
+    if args.ivar:
+        ivarConsensusFile = join(tempdir, 'temporary-consensus')
+        result = e.execute(
+            "samtools mpileup -A -Q 0 %s | "
+            "ivar consensus -p %s -q 20 -t %s -m %s" % (
```

Comment:
It's better to use `%r` for these. Then they will be in single quotes, which prevents any shell meta characters of spaces in file names from causing a problem.

Reply:
Also, I am usually strict with myself on things like this `-t %s`. Seeing as you know you are supposed to have a float, use `-t %f`.  That way if you somehow one day don't have a float, the code will break.